### PR TITLE
user_management models.py

### PR DESCRIPTION
**Pull request by CainanSquibb**
updated user_management models

### CODE REVIEW

**Review by julianbartosz — COMMENTED** [review]
Switch to using AbstractBaseUser for username==email. Looks good.

**Review by Obaideh001 — APPROVED** [review]

File: user_management/models.py [hunk 1]
```diff
@@ -1,22 +1,68 @@
...
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_superuser', True)
+
+        if extra_fields.get('is_superuser') is not True:
+            raise ValueError(_('Superuser must have is_superuser=True.'))
+
+        return self.create_user(email, password, **extra_fields)
+
+
+class User(AbstractBaseUser, PermissionsMixin):
```
Comment (elgin0) [inline user_management/models.py:25]: Great use of AbstractBaseUser.

**Review by elgin0 — APPROVED** [review]
Code looks good.

**Review by Axvelez — APPROVED** [review]
can you explain what gettext_lazy as _ does

**Pull request merged**